tests/embedding/test_dyna.py

```diff
@@ -0,0 +1,111 @@
+# Copyright (C) 2022 - 2024 ANSYS, Inc. and/or its affiliates.
+# SPDX-License-Identifier: MIT
+#
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in all
+# copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+# SOFTWARE.
+
+"""LSDyna analysis test"""
+
+import os
+
+import pytest
+
+try:
+    from ansys.mechanical.core import global_variables
+except:
+    # No embedding - this import breaks test collection
+    global_variables = {}
+
+from .test_qk_eng_wb2 import get_assets_folder
+
+
+@pytest.mark.embedding
+@pytest.mark.windows_only
+def test_lsdyna(printer, embedded_app):
+    """
+    Unit test for LSDyna.
+
+    This test focuses on LSDyna analysis in Mechanical.
+    Runs only in Windows environment. This test involves
+    a simple geometry with high velocity
+    hitting on rigid wall.
+    """
```

Comment:
This test focuses on a LSDyna analysis in Mechanical for the Windows environment only. This test involves a simple geometry with high velocity striking a rigid wall.